cvat/schema.yml

```diff
@@ -1049,6 +1049,176 @@ paths:
       responses:
         '204':
           description: The comment has been deleted
+  /api/consensus/merges:
+    post:
+      operationId: consensus_create_merge
+      summary: Create a consensus merge
+      parameters:
+      - in: query
+        name: rq_id
+        schema:
+          type: string
+        description: |
+          The consensus merge request id. Can be specified to check operation status.
```

Comment:
We should be using the request API for status checking.

Reply:
Currently, the reason is that Requests API v2 is not implemented yet. It's hard to integrate anything except import or export into the same server code. Supporting other use cases is one of the key goals of v2.

Reply:
This will create one more API style. This one is used for quality reports, job validations, and analytics.